packages/ibm-products/src/components/InterstitialScreen/InterstitialScreen.test.js

```diff
@@ -172,4 +179,52 @@ describe(componentName, () => {
       componentName
     );
   });
+
+  it('clicking on the next and back button', async () => {
+    renderComponent({
+      className: blockClass,
+      interstitialAriaLabel: 'Modal Interstitial Screen',
+    });
+    expect(screen.getByText('Next'));
+    expect(screen.getByText('Step 1'));
+    const step1 = screen.getByText('Step 1');
+    const listElement1 = step1.closest('li');
+    const step2 = screen.getByText('Step 2');
+    const listElement2 = step2.closest('li');
+    expect(listElement1).toHaveClass(
+      `${carbon.prefix}--progress-step--current`
+    );
+    expect(listElement2).toHaveClass(
+      `${carbon.prefix}--progress-step--incomplete`
+    );
+    const nextButtonElement = screen.getByText('Next');
+    expect(nextButtonElement).toHaveClass(`${blockClass}--next-btn`);
+    await act(() => userEvent.click(nextButtonElement));
+    expect(listElement1).toHaveClass(
+      `${carbon.prefix}--progress-step--complete`
+    );
+    expect(listElement2).toHaveClass(
+      `${carbon.prefix}--progress-step--current`
+    );
+    expect(screen.getByText('Back')).toBeInTheDocument();
+    const backButtonElement = screen.getByText('Back');
+    expect(backButtonElement).toHaveClass(`${blockClass}--prev-btn`);
+    await act(() => userEvent.click(backButtonElement));
+    expect(listElement1).toHaveClass(
+      `${carbon.prefix}--progress-step--current`
+    );
+    expect(listElement2).toHaveClass(
+      `${carbon.prefix}--progress-step--incomplete`
+    );
+  });
+
+  it('Clicking the close button', async () => {
+    renderComponent({
+      className: blockClass,
+      interstitialAriaLabel: 'Modal Interstitial Screen',
+    });
+    const closeBtn = screen.getByLabelText('Close');
+    await act(() => userEvent.click(closeBtn));
+    expect(onClose).toBeCalled();
```

Comment:
Just a note, you could use `.toHaveBeenCalledTimes(1)` here to make sure it's called the number of times expected.